### PR TITLE
Bug/sscs 2437 allow new lines

### PR DESCRIPTION
**Pull request by iguannago**

### CODE REVIEW

File: utils/regex.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const niNumber  = /^\s*[a-zA-Z]{1}\s*[a-zA-Z]{1}\s*(?:\s*\d\s*){6}[a-zA-Z]{1}\s*
 const title = /^[a-zA-Z /&]{2,}$/;
 const firstName = /^[a-zA-z]+([-][a-zA-Z]+)*$/;
 const lastName  = /^[a-zA-z]+([ '-][a-zA-Z]+)*$/;
-const whitelist = /^[a-zA-Z0-9 .",'?!()£:-]{2,}$/;
+const whitelist = /^[a-zA-Z0-9 \r\n.",'?!()£:-]{2,}$/;
```
Comment (bendiggle) [inline utils/regex.js]: Just a Question for everyone (@iguannago @paulgain ) Should we maybe create a new regex similar to this for text areas, as we probably don't want to allow new lines in text fields?

Comment (iguannago) [reply utils/regex.js]: that's a good point

File: test/e2e/scenarios/reasons-for-appealing/reasonsForAppealing.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    I.createTheSession();
+});
+
+After((I) => {
+    I.endTheSession();
+});
+
+Scenario ('Textfield allows to enter new lines', (I) => {
+
+    I.amOnPage(paths.reasonsForAppealing.reasonForAppealing);
```
Comment (paulgain) [inline test/e2e/scenarios/reasons-for-appealing/reasonsForAppealing.js]: For consistency we always put the `I.amOnPage` directly after `I.createTheSession();`

File: test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                cya: {
+                    reasonForAppealing: {
+                        question: 'my question'
+                    }
+                }
+            };
+
+        });
+
+        it('.section', () => {
```
Comment (paulgain) [inline test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js]: BDD Style. Can we please start with `it('should...')`. In needs to be expressive and helpful. Thanks.

File: test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            expect(reasonForAppealing.answers()[0].section).to.eql('reasons-for-appealing');
+        });
+
+        it('.answer', () => {
```
Comment (paulgain) [inline test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js]: Same as above

File: test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+            expect(reasonForAppealing.answers()[0].answer).to.eql('my answer');
+        });
+
+        it('.question', () => {
```
Comment (paulgain) [inline test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js]: Same as above.

File: test/e2e/scenarios/reasons-for-appealing/reasonsForAppealing.js (hunk 2 above)
Comment (iguannago) [reply test/e2e/scenarios/reasons-for-appealing/reasonsForAppealing.js]: done

File: test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js (hunk 3 above)
Comment (iguannago) [reply test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js]: done

File: test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js (hunk 4 above)
Comment (iguannago) [reply test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js]: done

File: test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js (hunk 5 above)
Comment (iguannago) [reply test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js]: done

File: test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+                }
+            };
+
+        });
+
+        it('should return expected section', () => {
+            expect(reasonForAppealing.answers()[0].section).to.eql('reasons-for-appealing');
+        });
+
+        it('should retrun expected answer', () => {
```
Comment (paulgain) [inline test/unit/steps/reasons-for-appealing/ReasonForAppealling.test.js]: Typo

**Pull request merged**